src/TicTacToe.elm

```diff
@@ -24,18 +25,25 @@ update : Msg -> Model -> Model
 update msg model =
   case msg of
     TakeTurn position ->
-      takeTurn position model
+      case model.playState of
+        InProgress model -> {playState = InProgress (takeTurn position model)}
+        NotStarted -> {playState = NotStarted}
```

Comment:
Is there a possibility where we can receive a `TakeTurn` instance `Msg` when the model is `NotStarted`?

The `view` doesn't seem to render anything that will trigger a `TakeTurn` when `NotStarted`. I am not sure if it will be triggered in the future.